resources/definitions/fdmprinter.def.json

```diff
@@ -3070,7 +3235,32 @@
                     "default_value": false,
                     "limit_to_extruder": "support_interface_extruder_nr",
                     "enabled": "support_enable",
-                    "settable_per_mesh": true
+                    "settable_per_mesh": true,
+                    "children":
+                    {
+                        "support_bottom_enable":
+                        {
+                            "label": "Enable Support Bottom",
+                            "description": "Generate a dense flooring below the support where it rests on the model. This will create a skin between the model and support.",
```

Comment:
I've rephrased "flooring" to "floors".

I haven't changed the other wording because I don't think how a user would look at it, not knowing how support bottoms are created.